programs/github-pull-request/spec.js

```diff
@@ -1,31 +1,50 @@
-const DEFAULT_ARGS = {
+const APP_ARGS = {
+	appId: 'APP_ID',
+	appPrivateKey: 'APP_PRIVATE_KEY',
+};
+
+const PULL_ARGS = {
 	token: 'TOKEN',
 	user: 'USER',
 	repo: 'REPO',
 	pr: 'PR',
 };
 
+const DEFAULT_ARGS = {
+	...PULL_ARGS,
+	...APP_ARGS,
+};
+
 describe('github-pull-request', () => {
-	let args;
+	let argsPull;
+	let argsApp;
 	let githubPullRequest;
 	before(() => {
 		require('./pull');
-		require.cache[require.resolve('./pull')].exports = _args => { args = _args; };
+		require('./getGithubAppUserToken');
+		require.cache[require.resolve('./pull')].exports = _args => { argsPull = _args; };
```

Comment:
Why the hek we manipulate require caching instead of mocking stuff? 🤔

Reply:
I don't know I didn't want to refactor it now

Reply:
🤷‍♂️